File: src/main/java/com/lawencon/elearn/dto/classdtl/ClassDtlDataDto.java
package com.lawencon.elearn.dto.classdtl;

public class ClassDtlDataDto {
	private Long id;
	private Long classesId;
	private String classesCode;
	private String classesSubject;
	private String classesDescription;
	private Long instructorId;
	private String email;
	private String fullname;
	private Integer ver;
	private Boolean isActive;
	private Long fileId;
	
	public String getClassesDescription() {
		return classesDescription;
	}
	public void setClassesDescription(String classesDescription) {
		this.classesDescription = classesDescription;
	}
	public Boolean getIsActive() {
		return isActive;
	}
	public void setIsActive(Boolean isActive) {
		this.isActive = isActive;
	}
	public Long getFileId() {
		return fileId;
	}
	public void setFileId(Long fileId) {
		this.fileId = fileId;
	}
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public Long getClassesId() {
		return classesId;
	}
	public void setClassesId(Long classesId) {
		this.classesId = classesId;
	}
	public String getClassesCode() {
		return classesCode;
	}
	public void setClassesCode(String classesCode) {
		this.classesCode = classesCode;
	}
	public String getClassesSubject() {
		return classesSubject;
	}
	public void setClassesSubject(String classesSubject) {
		this.classesSubject = classesSubject;
	}
	public Long getInstructorId() {
		return instructorId;
	}
	public void setInstructorId(Long instructorId) {
		this.instructorId = instructorId;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public String getFullname() {
		return fullname;
	}
	public void setFullname(String fullname) {
		this.fullname = fullname;
	}
	public Integer getVer() {
		return ver;
	}
	public void setVer(Integer ver) {
		this.ver = ver;
	}
}

File: frontend/src/app/dto/class-dtl/class-dtl.data.dto.ts
export class ClassDtlDataDto{
    private _id! : number
	private _classesId! : number
	private _classesCode! : string
	private _classesSubject! : string
    private _classesDescription! : string
	private _instructorId! : number
	private _email! : string
	private _fullname! : string
	private _ver! : number
	private _isActive! : boolean
	private _fileId! : number

    public get classesDescription(): string {
        return this._classesDescription;
    }

    public set classesDescription(classesDescription: string) {
        this._classesDescription = classesDescription;
    }

    public get id(): number {
        return this._id;
    }

    public set id(id: number) {
        this._id = id;
    }

    public get classesId(): number {
        return this._classesId;
    }

    public set classesId(classesId: number) {
        this._classesId = classesId;
    }

    public get classesCode(): string {
        return this._classesCode;
    }

    public set classesCode(classesCode: string) {
        this._classesCode = classesCode;
    }

    public get classesSubject(): string {
        return this._classesSubject;
    }

    public set classesSubject(classesSubject: string) {
        this._classesSubject = classesSubject;
    }

    public get instructorId(): number {
        return this._instructorId;
    }

    public set instructorId(instructorId: number) {
        this._instructorId = instructorId;
    }

    public get email(): string {
        return this._email;
    }

    public set email(email: string) {
        this._email = email;
    }

    public get fullname(): string {
        return this._fullname;
    }

    public set fullname(fullname: string) {
        this._fullname = fullname;
    }

    public get ver(): number {
        return this._ver;
    }

    public set ver(ver: number) {
        this._ver = ver;
    }

    public get isActive(): boolean {
        return this._isActive;
    }

    public set isActive(isActive: boolean) {
        this._isActive = isActive;
    }

    public get fileId(): number {
        return this._fileId;
    }

    public set fileId(fileId: number) {
        this._fileId = fileId;
    }

    


}